.gitignore

```diff
@@ -6,3 +6,5 @@ unpacked
 _meta/kibana/index-pattern
 
 *.pyc
+*.iml
+.idea/*
```

Comment:
A nice trick it to add such non-project-specific ignores to your 'global gitignore' - some useful templates can be found at https://github.com/github/gitignore .

But it can't hurt to have them here, too :)